### DISCUSSION TITLE
TST disabled with new window

### DISCUSSION THREAD

**Discussion by paulri** [piroor/treestyletab#3092] [Q&A]
When I turn a new tab into a new window, or go to the taskbar in Windows and select the option to make a new window, the new window that is created does not have TST enabled. I have to go to the toolbar and click on the TST icon. 

Is there a way to make sure that TST is always enabled, so I don't have to click to engage it? I don't ever create new windows of my browser not wanting TST enabled.

**Comment by irvinm** [comment]
Is the new window a private window or just a regular window?  If private, TST has to be granted access to work in a private window.

**Reply by paulri** [reply]
No, it is a regular window.

**Reply by irvinm** [reply]
Just checking, but are you sure it is "disabled" vs. just "hidden"?  If you make the sidebar visible (F1), do you see TST active there or at least in the list of items in the sidebar?

**Reply by paulri** [reply]
I'll report back when I can reproduce the problem. Of course, after you replied, I checked and TST creates a vertical tab bar when I make a new tab. :(

**Reply by paulri** [reply]
Well I finally saw it--the last few days whenever I've made a new window, it has appeared. 

Just now, when creating a new window, TST was not displayed. And after pressing F1, TST was displayed.

**Comment by piroor** [comment]
Sorry it is uncontrollable from TST.

* The initial state of the sidebar for a new window is controlled by Firefox.
* There is an API `browser.sidebarAction.open()` to ensure Firefox to open TST sidebar, but it is callable only on very limited situations.

Thus it looks need to be reported to Firefox's bug tracker as a request to make it possible from addons.
